x-pack/plugins/ingest_manager/common/openapi/spec_oas3.json

```diff
@@ -1280,9 +1280,9 @@
         ]
       }
     },
-    "/fleet/setup": {
+    "/fleet/agents/setup": {
```

Comment:
```suggestion
    "/agents/setup": {
```

We need to drop the leading "/fleet" from the other paths in this file as well. Our root is `http://localhost:5601/api/fleet` which would make this `http://localhost:5601/api/fleet/fleet/agents/setup`

If you load the PR spec file in a viewer like  https://mrin9.github.io/OpenAPI-Viewer/#/load/https%3A%2F%2Fraw.githubusercontent.com%2Felastic%2Fkibana%2F9410bc780cff94971727c36627419483c53a102b%2Fx-pack%2Fplugins%2Fingest_manager%2Fcommon%2Fopenapi%2Fspec_oas3.json it is easier to see.

PR on left, changes on right
<img width="1310" alt="Screen Shot 2020-10-02 at 9 40 17 AM" src="https://user-images.githubusercontent.com/57655/94929651-61a77480-0493-11eb-8141-a05aded59826.png">

Notice the URLs the client tries to reach
<img width="1513" alt="Screen Shot 2020-10-02 at 9 45 32 AM" src="https://user-images.githubusercontent.com/57655/94930232-19d51d00-0494-11eb-8381-2163a29a51d1.png">